Mathlib/Algebra/Algebra/Subalgebra/Basic.lean

```diff
@@ -626,14 +626,12 @@ def rangeRestrict (f : A →ₐ[R] B) : A →ₐ[R] f.range :=
   f.codRestrict f.range f.mem_range_self
 #align alg_hom.range_restrict AlgHom.rangeRestrict
 
-theorem rangeRestrict_surjective (f : A →ₐ[R] B):
-    Function.Surjective (f.rangeRestrict) :=
-  fun ⟨_y, hy⟩ =>
+theorem rangeRestrict_surjective (f : A →ₐ[R] B) :
+    Function.Surjective (f.rangeRestrict) := fun ⟨_y, hy⟩ =>
```

Comment:
I'm not convinced by this one: I'd rather have the proof cleanly separated from the statement, unless it fits on one single line.